cmd/crictl/info.go

```diff
@@ -17,44 +17,75 @@ limitations under the License.
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/ghodss/yaml"
 	"github.com/urfave/cli"
 	"golang.org/x/net/context"
 	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
-const (
-	criClientVersion = "v1alpha1"
-)
-
-var runtimeVersionCommand = cli.Command{
-	Name:  "info",
-	Usage: "Display runtime version information",
+var runtimeStatusCommand = cli.Command{
+	Name:      "info",
+	Usage:     "Display information of the container runtime",
+	ArgsUsage: "",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "output, o",
+			Value: "json",
+			Usage: "Output format, One of: json|yaml",
+		},
+	},
 	Action: func(context *cli.Context) error {
-		err := Version(runtimeClient, criClientVersion)
+		err := Info(context, runtimeClient)
 		if err != nil {
-			return fmt.Errorf("Getting the runtime version failed: %v", err)
+			return fmt.Errorf("getting status of runtime failed: %v", err)
 		}
 		return nil
 	},
 	Before: getRuntimeClient,
 	After:  closeConnection,
 }
 
-// Version sends a VersionRequest to the server, and parses the returned VersionResponse.
-func Version(client pb.RuntimeServiceClient, version string) error {
-	request := &pb.VersionRequest{Version: version}
-	logrus.Debugf("VersionRequest: %v", request)
-	r, err := client.Version(context.Background(), request)
-	logrus.Debugf("VersionResponse: %v", r)
+// Info sends a StatusRequest to the server, and parses the returned StatusResponse.
+func Info(cliContext *cli.Context, client pb.RuntimeServiceClient) error {
+	request := &pb.StatusRequest{}
+	logrus.Debugf("StatusRequest: %v", request)
+	r, err := client.Status(context.Background(), request)
+	logrus.Debugf("StatusResponse: %v", r)
+	if err != nil {
+		return err
+	}
+
+	statusByte, err := json.Marshal(r.Status)
```

Comment:
We can make this a helper function, because we'll need it for container/sandbox/image status.
I'm fine with doing that in another PR.

Reply:
Will do late. :)